src/Sql/Sql.sqlproj

```diff
@@ -274,5 +274,18 @@
     <Build Include="dbo\Stored Procedures\User_ReadBySsoUserOrganizationIdExternalId.sql" />
     <Build Include="dbo\Stored Procedures\SsoUser_Update.sql" />
     <Build Include="dbo\Stored Procedures\SsoUser_ReadById.sql" />
+    <Build Include="dbo\Stored Procedures\Cipher_DeleteByIdsOrganizationId.sql" />
+    <Build Include="dbo\Stored Procedures\Cipher_SoftDeleteByIdsOrganizationId.sql" />
+    <Build Include="dbo\Stored Procedures\Organization_ReadByIdentifier.sql" />
+    <Build Include="dbo\Stored Procedures\Send_Create.sql" />
+    <Build Include="dbo\Stored Procedures\Send_DeleteById.sql" />
+    <Build Include="dbo\Stored Procedures\Send_ReadById.sql" />
+    <Build Include="dbo\Stored Procedures\Send_ReadByUserId.sql" />
+    <Build Include="dbo\Stored Procedures\Send_Update.sql" />
+    <Build Include="dbo\Stored Procedures\SsoConfig_ReadManyByNotBeforeRevisionDate.sql" />
```

Comment:
`SsoConfig_...`? Seems off. Was this missing from the project originally (which would have been my fault of course)?

Reply:
I added a bunch of missing files to the project. I think the other guys that are not using VS keep adding them without being in the project.

Reply:
I'm also guilty, will get better at that in other PR reviews as well.